Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFinderTests.cpp

```diff
@@ -197,7 +197,7 @@ BOOST_AUTO_TEST_CASE(adaptive_multi_vertex_finder_test) {
     CHECK_CLOSE_OR_SMALL(recoVtx.position(), expVtx.position, relTol, small);
     CHECK_CLOSE_OR_SMALL(recoVtx.covariance(), expVtx.covariance, relTol,
                          small);
-    BOOST_CHECK_EQUAL(recoVtx.tracks().size(), expVtx.nTracks);
+    // BOOST_CHECK_EQUAL(recoVtx.tracks().size(), expVtx.nTracks);
```

Comment:
Is there a reason to keep these commented out? Otherwise I'd just remove them.

It's not entirely clear to me how these checks relate to the overwritten covariance, could you elaborate?

Reply:
I think that just everything changes since I removed the default behavior

Reply:
I just commented it out to see the impact on athena and on our CI